VirtualBox COM Server 5.1.26 r117224 win.amd64 (Jul 27 2017 12:32:13) release log
00:00:00.101689 main     Log opened 2017-12-24T11:18:10.871411600Z
00:00:00.101689 main     Build Type: release
00:00:00.101689 main     OS Product: Windows 8.1
00:00:00.101689 main     OS Release: 6.3.9600
00:00:00.101689 main     OS Service Pack: 
00:00:00.255740 main     DMI Product Name: Inspiron 5537
00:00:00.271360 main     DMI Product Version: A07
00:00:00.271360 main     Host RAM: 8072MB (7.8GB) total, 6674MB (6.5GB) available
00:00:00.271360 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.271360 main     Process ID: 1744
00:00:00.271360 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.286986          VirtualBox: object creation starts
00:00:00.286986          Home directory: 'C:\Users\Ulhas\.VirtualBox'
00:00:00.286986          Installed Drivers:
00:00:00.286986            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 5.1.26.17224)
00:00:00.286986            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 5.1.26.17224)
00:00:00.286986            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 5.1.26.17224)
00:00:00.302612            C:\WINDOWS\system32\DRIVERS\VBoxNetAdp6.sys (Version: 5.1.26.17224)
00:00:00.302612          Loading settings file "C:\Users\Ulhas\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.417245          Getting USB descriptor failed with error 31
00:00:00.417245          USB: Unknown USB device detected (idVendor: 0x0c45, idProduct: 0x64ad)
00:00:00.417245          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x07dc)
00:00:00.464156          HostDnsMonitor: old information
00:00:00.464156            no server entries
00:00:00.464156            no domain set
00:00:00.464156            no search string entries
00:00:00.464156          HostDnsMonitor: new information
00:00:00.464156            server 1: 192.168.43.1
00:00:00.464156            no domain set
00:00:00.464156            no search string entries
00:00:00.464156          HostDnsMonitorProxy::notify
00:00:00.636034          VD: VDInit finished
00:00:00.636034          Loading settings file "C:\Users\Ulhas\VirtualBox VMs\Ansible_Control_Machine\Ansible_Control_Machine.vbox" with version "1.16-windows"
00:00:00.636034          Loading settings file "C:\Users\Ulhas\VirtualBox VMs\Ansible_node_1\Ansible_node_1.vbox" with version "1.16-windows"
00:00:00.651659          Loading settings file "C:\Users\Ulhas\VirtualBox VMs\Ansible_Node_2\Ansible_Node_2.vbox" with version "1.16-windows"
00:00:00.651659          VirtualBox: object created
00:01:07.231410          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={edba9d10-45d8-b440-1712-46ac0c9bc4c5} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:01:07.350800          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={b2547866-a0a1-4391-8b86-6952d82efaa0} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=0
00:01:19.581856          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={edba9d10-45d8-b440-1712-46ac0c9bc4c5} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:01:19.670816          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={b2547866-a0a1-4391-8b86-6952d82efaa0} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=0
00:01:28.345223          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={edba9d10-45d8-b440-1712-46ac0c9bc4c5} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:01:28.388239          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={b2547866-a0a1-4391-8b86-6952d82efaa0} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=0
01:21:49.784869 USBPROXY Getting USB descriptor failed with error 31
01:21:49.968403 MainPower Host suspending: Paused 3 VMs
02:09:03.551800 MainPower Host resumed: Resumed 3 VMs
02:09:04.054291 USBPROXY Getting USB descriptor failed with error 31
02:09:19.171132 USBPROXY Getting USB descriptor failed with error 31
03:07:39.048955 USBPROXY Getting USB descriptor failed with error 31
03:08:08.375723 dns-monitor HostDnsMonitor: old information
03:08:08.375723 dns-monitor   server 1: 192.168.43.1
03:08:08.375723 dns-monitor   no domain set
03:08:08.375723 dns-monitor   no search string entries
03:08:08.375723 dns-monitor HostDnsMonitor: new information
03:08:08.375723 dns-monitor   no server entries
03:08:08.375723 dns-monitor   no domain set
03:08:08.375723 dns-monitor   no search string entries
03:08:08.421613 dns-monitor HostDnsMonitorProxy::notify
03:08:12.714263 USBPROXY Getting USB descriptor failed with error 31
03:08:21.320768 USBPROXY Getting USB descriptor failed with error 31
03:08:21.324177 USBPROXY Getting USB descriptor failed with error 31
03:17:16.543292 dns-monitor HostDnsMonitor: old information
03:17:16.543292 dns-monitor   no server entries
03:17:16.543292 dns-monitor   no domain set
03:17:16.543292 dns-monitor   no search string entries
03:17:16.543292 dns-monitor HostDnsMonitor: new information
03:17:16.543292 dns-monitor   server 1: 192.168.43.1
03:17:16.543292 dns-monitor   no domain set
03:17:16.543292 dns-monitor   no search string entries
03:17:16.543292 dns-monitor HostDnsMonitorProxy::notify
03:17:26.371713 USBPROXY Getting USB descriptor failed with error 31
03:18:12.058289 USBPROXY Getting USB descriptor failed with error 31
03:22:57.301196 USBPROXY Getting USB descriptor failed with error 31
03:24:30.195003 USBPROXY Getting USB descriptor failed with error 31
03:24:50.149953 USBPROXY Getting USB descriptor failed with error 31
03:24:50.150902 USBPROXY Getting USB descriptor failed with error 31
03:24:50.150902 USBPROXY Getting USB descriptor failed with error 31
03:26:45.262971 USBPROXY Getting USB descriptor failed with error 31
03:26:48.346206 MainPower Host suspending: Paused 3 VMs
03:59:37.867066 dns-monitor HostDnsMonitor: old information
03:59:37.867066 dns-monitor   server 1: 192.168.43.1
03:59:37.867066 dns-monitor   no domain set
03:59:37.867066 dns-monitor   no search string entries
03:59:37.867066 dns-monitor HostDnsMonitor: new information
03:59:37.867066 dns-monitor   no server entries
03:59:37.867066 dns-monitor   no domain set
03:59:37.867066 dns-monitor   no search string entries
03:59:37.867066 dns-monitor HostDnsMonitorProxy::notify
03:59:37.983405 MainPower Host resumed: Resumed 3 VMs
04:00:56.424178 USBPROXY Getting USB descriptor failed with error 31
04:00:57.841707 dns-monitor HostDnsMonitor: old information
04:00:57.841707 dns-monitor   no server entries
04:00:57.841707 dns-monitor   no domain set
04:00:57.841707 dns-monitor   no search string entries
04:00:57.841707 dns-monitor HostDnsMonitor: new information
04:00:57.841707 dns-monitor   server 1: 192.168.43.1
04:00:57.841707 dns-monitor   no domain set
04:00:57.841707 dns-monitor   no search string entries
04:00:57.841707 dns-monitor HostDnsMonitorProxy::notify
04:14:41.510041          ERROR [COM]: aRC=VBOX_E_INVALID_VM_STATE (0x80bb0002) aIID={b2547866-a0a1-4391-8b86-6952d82efaa0} aComponent={SessionMachine} aText={The machine is not mutable or saved (state is Stopping)}, preserve=false aResultDetail=0
04:14:41.523729          ERROR [COM]: aRC=VBOX_E_INVALID_VM_STATE (0x80bb0002) aIID={b2547866-a0a1-4391-8b86-6952d82efaa0} aComponent={SessionMachine} aText={The machine is not mutable or saved (state is Stopping)}, preserve=false aResultDetail=0
04:14:55.405820 main     VirtualBox: object deletion starts
04:14:55.437098 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={4afe423b-43e0-e9d0-82e8-ceb307940dda} aComponent={MediumWrap} aText={Medium 'C:\Users\Ulhas\VirtualBox VMs\Ansible_Control_Machine\Ansible_Control_Machine.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
04:14:55.452720 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={4afe423b-43e0-e9d0-82e8-ceb307940dda} aComponent={MediumWrap} aText={Medium 'C:\Users\Ulhas\VirtualBox VMs\Ansible_node_1\Ansible_node_1.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
04:14:55.452720 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={4afe423b-43e0-e9d0-82e8-ceb307940dda} aComponent={MediumWrap} aText={Medium 'C:\Users\Ulhas\VirtualBox VMs\Ansible_Node_2\Ansible_Node_2.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
04:14:55.593347 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={0169423f-46b4-cde9-91af-1e9d5b6cd945} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
04:14:55.608972 main     VirtualBox: object deleted
